index.html

```diff
@@ -271,19 +270,26 @@ <h3 class="connect">LET'S CONNECT</h3>
               </li>
               <hr>
             </ul>
-            
-            
-
-            
-            
-              
-
-                  
-                  
-                
-                  
+              <!-------------------------------------- Contact form------------------------------->
+            <div class="about-contact">
+              <div class="title-container">
+                <h2 class="contact">Contact Me</h2>
+                <p class="para">If you have an application you are interested in<br>developing, 
+                  a feature that you need built or a project<br>that needs coding. i'd love to help with it</p>
+              </div>
+              <div class="form-container">
+                <form action="https://formspree.io/f/{form_id}" method="post" class="labels">
+                    <input type="text" name="name" id="name" placeholder="enter name" required>
+                    <input type="email" name="email" id="email" placeholder="enter email" required>
+                    <textarea name="message" id="message" cols="30" rows="10" placeholder="message" required></textarea>
```

Comment:
- [ ] Name input should have a `maxlength="30"` attribute.
- [ ] The  Message input should have a  `maxlength="500"` attribute.
- This is right to give proper validations 👍